packages/hub/test/scope.test.ts

```diff
@@ -63,10 +63,26 @@ describe('Scope', () => {
 
     test('addBreadcrumb', () => {
       const scope = new Scope();
-      scope.addBreadcrumb({ message: 'test' }, 100);
+      scope.addBreadcrumb({ message: 'test' });
       expect((scope as any)._breadcrumbs[0]).toHaveProperty('message', 'test');
     });
 
+    test('addBreadcrumb can be limited to hold up to N breadcrumbs', () => {
+      const scope = new Scope();
+      for (let i = 0; i < 10; i++) {
+        scope.addBreadcrumb({ message: 'test' }, 5);
+      }
+      expect((scope as any)._breadcrumbs).toHaveLength(5);
```

Comment:
We could also specifically check for which 5 breadcrumbs we want: last, first, etc. I think it should be last N, right?